example/integration_test/bucketeer_flutter_sdk_test.dart

```diff
@@ -351,4 +355,27 @@ void main() async {
           BKTClient.instance.removeEvaluationUpdateListener(listenToken);
         });
   });
+
+  test('BKTClient.initialize should allow feature_tag to be optional', () async {
+    final config = BKTConfigBuilder()
+        .apiKey(Constants.apiKey)
+        .apiEndpoint(Constants.apiEndpoint)
+        .debugging(debugging)
+        .eventsMaxQueueSize(Constants.exampleEventMaxQueueSize)
+        .eventsFlushInterval(Constants.exampleEventsFlushInterval)
+        .pollingInterval(Constants.examplePollingInterval)
+        .backgroundPollingInterval(Constants.exampleBackgroundPollingInterval)
+        .appVersion(appVersion)
+        .build();
+    final user = BKTUserBuilder().id(userId).customAttributes({}).build();
+
+    await BKTClient.initialize(
+      config: config,
+      user: user,
+    ).onError((error, stackTrace) => fail("initialize() should success"));
```

Comment:
We should also ensure that are getting the evaluations from the server correctly.
Use some get variation interface to check it.